x-pack/plugins/reporting/server/routes/internal/management/integration_tests/jobs.test.ts

```diff
@@ -388,4 +409,58 @@ describe('GET /api/reporting/jobs/download', () => {
         );
     });
   });
+
+  describe('usage counters', () => {
```

Comment:
Added more usage counter jest integration tests since the API integration tests have been taken out.